inference-engine/tests/functional/inference_engine/conditional_compilation/selective_build.cpp

```diff
@@ -76,6 +76,24 @@ TEST(ConditionalCompilationTests, SimpleScope) {
 #undef CCTests_Scope0
 }
 
+TEST(ConditionalCompilationTests, SimpleSectionDefine) {
+#define CCTests_Section1 1
+
+    int n = 0;
+#if OV_SCOPE_DEFINE(CCTests, Section1)
+    OV_SCOPE_TASK(CCTests, Section1);
```

Comment:
Not a good solution, because we should copy and paste the same line. If we make a mistake on the second line, the annotated block will always be disabled.

Reply:
Yes, but I didn't invented how to do it in one line... because I cannot create itt counter inside the macro which should return true or false.

Reply:
But the advantage of this proposal is that I can use this macro if an ITT counter and the code, which we want to disable,  are located in different places.